NGF: Update all existing guides to use cert-manager to manage secrets (#1933)
nginx/documentation: content/ngf/traffic-management/tls-passthrough.md

## Set up

{{< include "ngf/deploy-cert-manager.md" >}}

{{< include "ngf/cert-manager-local-ca.md" >}}

> Create a `Certificate` for `app.example.com`. cert-manager will create the `app-tls-secret` Secret (containing `tls.crt`, `tls.key`, and `ca.crt`) that the `secure-app` Pod mounts:

Reviewer: ```suggestion
Create a `Certificate` for `app.example.com`. cert-manager creates the `app-tls-secret` Secret, which contains `tls.crt`, `tls.key`, and `ca.crt` and is mounted by the `secure-app` Pod:
```

{{< call-out "note" >}}If you have a DNS record allocated for `app.example.com`, you can send the request directly to that hostname, without needing to resolve.{{< /call-out >}}

> Send a request to the `secure-app` Service on the TLS port with the `--insecure` flag. The `--insecure` flag is required because the `secure-app` is using a certificate signed by a local self-signed CA that curl does not trust.

Reviewer: ```suggestion
Send a request to the `secure-app` Service on the TLS port with the `--insecure` flag. The flag is required because `secure-app` uses a certificate signed by a local self-signed CA that curl does not trust.
```
